### DISCUSSION TITLE
idf.py can't works

### DISCUSSION THREAD

**Discussion by Mowitra** [Mair/esp32-course#143] [Course Qeustions] [answered]
sir why it shows this type of errors

Install the latest PowerShell for new features and improvements! https://aka.ms/PSWindows

PS C:\ESP_IDF\eg1> idf.py
idf.py : The term 'idf.py' is not recognized as the name of a cmdlet, function, script file, or operable program. Check the spelling of the name, or   
if a path was included, verify that the path is correct and try again.
At line:1 char:1
+ idf.py
+ ~~~~~~
    + CategoryInfo          : ObjectNotFound: (idf.py:String) [], CommandNotFoundException
    + FullyQualifiedErrorId : CommandNotFoundException

**Comment by Mair** [comment, marked as answer]
try the IDF termainal
